studio/schemas/post.js

```diff
@@ -38,10 +32,10 @@ export default {
       },
     },
     {
-      name: 'categories',
-      title: 'Categories',
+      name: 'subtag',
+      title: 'Subtag',
       type: 'array',
-      of: [{ type: 'reference', to: { type: 'category' } }],
+      of: [{ type: 'reference', to: { type: 'subTag' } }],
```

Comment:
I think location is an attribute of carparks instead for posts
and tags and subtags can also be applied to posts as well